### DISCUSSION TITLE
Welcome to TWCManager Discussions!

### DISCUSSION THREAD

**Discussion by ngardiner** [ngardiner/TWCManager#408] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TitusJohannes** [comment]
Hello, I am using TWCManager (latest version) for quire a while. Since few days it stopped working, to be more precise, it does not start and stop charging via API. However, the tokens are OK and the API in general works fine (e.g. if I send the command "Debug" --> { "charge_rate": 5 } after ~5 seconds I see it in the Tesla App. The start and stop command will work only if I set in the "onlyChargeMultiCarsAtHome": false"
The "homeLat": XX.573428, "homeLon": XX.993959, is set correctly, is not changed, and was checked and verified via googlemaps.
Is there somebody with the same problem: I have a MS 2023.12.1.1 and MX 2023.6.11 with both cars, same problem.

THanks for any hint!

Yours sincerley,
Titus

**Reply by MikeBishop** [reply]
This is likely the same as #504.

**Comment by TitusJohannes** [comment]
Hello MIke,
yes thats obviously the same problem. Thanks for letting me know...

Yours Titus
